source/items/buildingtool.md

```diff
@@ -59,15 +59,15 @@ To place a worker hut or a building from the mod, you first need to have the hut
 - The green check mark is for when you are done and want to commit to placing it.
 - The red X is to cancel the whole process.
 
-- **Note:** When in creative mode you will also see in the lower right the *Copy* and *Paste* options.
+- **Note:** When in creative mode you will also see in the lower right, the *Copy* and *Paste* options.
 
 <br>
 
-<p style="color:Blue;"><b>Hint:</b> Once you have the GUI open you can press the ESC button on your keyboard so that the 3D image will remain and you can move around and take a better look at it from all sides to make sure it's the way you want it! Then you can right click with the building tool again towards the 3D image (on a block or even in the air) to go back to the GUI so you can "nudge" your structure to the best place and orientation that you want. You can repeat this process as many times as you want before clicking the green check mark to commit to the placement of the structure.</p>
+<p style="color:Blue;"><b>Hint:</b> Once you have the GUI open, you can press the ESC button on your keyboard so that the 3D image will remain and you can move around and take a better look at it from all sides to make sure it's the way you want it! Then you can right click with the building tool again towards the 3D image (on a block or even in the air) to go back to the GUI so you can "nudge" your structure to the best place and orientation that you want. You can repeat this process as many times as you want before clicking the green check mark to commit to the placement of the structure.</p>
```

Comment:
```suggestion
<p style="color:Blue;"><b>Hint:</b> Once you have the GUI open, you can press the ESC button on your keyboard so that the 3D image will remain. Then, you can move around and take a better look at it from all sides to make sure it's the way you want it! Right-click with the building tool again towards the 3D image (on a block or in the air) to go back to the GUI so you can "nudge" your structure to the place and orientation that you want. You can repeat this process as many times as you want before clicking the green check mark to commit to the placement of the structure.</p>
```